src/main/scala/no/ndla/auth/service/TokenService.scala

```diff
@@ -32,7 +32,7 @@ trait TokenService {
         ("iat" -> now) ~
         ("exp" -> expires))
 
-      TokenResponse(JsonWebToken(jwtHeader, jwtClaims, clientSecret), "bearer", expires)
+      TokenResponse(JsonWebToken(jwtHeader, jwtClaims, clientSecret), "bearer", AuthProperties.TokenValidityInSeconds)
```

Comment:
For ryddighet kunne jeg tenke meg at du trakk ut denne i en egen variabel også, slik at vi både har `expires` og `expires_in` som variable. Da blir det enda tydeligere at dette er bevisst.